src/locales/en-us/common/index.yaml

```diff
@@ -3,3 +3,10 @@ common:
   copySuccessText: 'Copied!'
   copyKubernetesText: 'Copy as Kubernetes'
   copyKubernetesShortText: 'as k8s'
+  status:
+    online: 'Online'
+    offline: 'Offline'
+    partiallyDegraded: 'Partially degraded'
+    partially_degraded: 'Partially degraded'
+    notAvailable: 'Information not available'
+    not_available: 'Information not available'
```

Comment:
**Question**: Should these start with a capital letter? We currently do a mix of both.

Reply:
Yeah I also wondered, me I'd say yes but not knowing exactly where they are used, but I figured I'd now just leave that for i18n folks to change in the YAML files if they want to, kinda not our decision anymore 🎉 . I'd do whatever feels right to you right now and then people who decide these things can change.